components/AuthModal/AuthModal.tsx

```diff
@@ -77,7 +77,7 @@ export const AuthModal = () => {
       <Transition appear show={isOpen} as={Fragment}>
         <Dialog
           as="div"
-          tw="relative z-10"
+          tw="relative z-[999]"
```

Comment:
can you make it z-50?

Reply:
It is going to be in the uppermost of the screen that's why I used the maximum value of the z-index so that in any case it will be at the top. we use z-[999] for components who will always at the top .

Reply:
z-50 will also works until you don't hardcode any other component like this. that's why sticking to default numbers that tailwind provides is recommended. but anyways we can look into it later